exchanges/okex/okex_test.go

```diff
@@ -1576,29 +1560,52 @@ func TestGetETTSettlementPriceHistory(t *testing.T) {
 
 // Websocket tests ----------------------------------------------------------------------------------------------
 
-// TestWsLogin API endpoint test
-func TestWsLogin(t *testing.T) {
-	TestSetRealOrderDefaults(t)
+// TestSendWsMessages Logic test
+// Attempts to subscribe to a channel that doesn't exist
+// Will log in if credentials are present
+func TestSendWsMessages(t *testing.T) {
+	TestSetDefaults(t)
 	if !websocketEnabled {
 		t.Skip("Websocket not enabled, skipping")
 	}
-	if !o.Websocket.IsConnecting() || !o.Websocket.IsConnected() {
-		o.Websocket.Connect()
+	var dialer websocket.Dialer
+	var err error
+	var ok bool
+	o.Websocket.TrafficAlert = make(chan struct{}, 99)
+	o.WebsocketConn, _, err = dialer.Dial(o.Websocket.GetWebsocketURL(),
+		http.Header{})
+	if err != nil {
+		t.Errorf("%s Unable to connect to Websocket. Error: %s",
```

Comment:
This is probably better as t.Fatal

Reply:
For some reason I thought t.Fatal killed all tests, not just the current one. I was incorrect. Will amend!